catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/fragment/RenameSpriteTest.java

```diff
@@ -213,6 +217,45 @@ private void renameSpriteTo(String newSpriteName) {
 				.check(doesNotExist());
 	}
 
+	@Test
+	private void spriteEqualBackgroundTest() {
```

Comment:
Also even after setting it to public the test fails if the whole file of tests is executed. It only works if you only execute this one test.